validator_client/src/block_service.rs

```diff
@@ -401,6 +411,7 @@ impl<T: SlotClock + 'static, E: EthSpec> BlockService<T, E> {
             .await
             .map_err(|e| BlockError::Recoverable(format!("Unable to sign block: {:?}", e)))?;
 
+        info!(log, "Publishing signed block");
```

Comment:
```suggestion
        info!(
            log,
            "Publishing signed block";
            "slot" => slot.as_u64(),
        );
```

This might help when grep-ing the logs for `slot: x`.